app/policies/company_policy.rb

```diff
@@ -1,5 +1,5 @@
 class CompanyPolicy < ApplicationPolicy
-
+  include PoliciesHelper
```

Comment:
not needed (see comment later in this class)

Reply:
I refactored an existing method from `company_policy.rb` in order to use it both there as well as in `address_policy.rb`.

Reply:
OK.  after discussion I'm ok with it.